index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcd34e45ea.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="flex-container outer-container">
+        <header class="flex-container">
+            <nav class="flex-container nav-bar">
+                <!--- Item to display logo -->
+                <div class="flex-item" id="logo">
+                    Welcome
+                </div>
```

Comment:
- [OPTIONAL] As suggested by the previous reviewer, it is for convention, semantics and better user experience reasons to make the logo on the website redirects to the homepage. That is the reason for asking you to wrap the text `Welcome` which serves as your logo at this moment inside anchor tag `<a>`  make it to redirect to the homepage whenever it is clicked. Kindly attend to it.